variables.tf

```diff
@@ -101,6 +101,12 @@ variable "create_security_group" {
   type        = bool
 }
 
+variable "add_ibm_cloud_internal_rules" {
```

Comment:
I think this needs to move next to `name `field in the security_group variables.